src/PackageAnalyzer.jl

```diff
@@ -11,6 +11,8 @@ using Git: Git
 using Downloads
 using Tar
 using CodecZlib
+using Legolas
```

Comment:
We need Legolas in scope due to https://github.com/beacon-biosignals/Legolas.jl/issues/91